common/pb/serverinfo_user.proto

```diff
@@ -1,3 +1,4 @@
+syntax = "proto2";
```

Comment:
This is what stops the warning messages from showing when you compile the .proto files.  it could use applied to all the .proto files.  I plan on putting a PR in to resolve all the ones that do not have the line in it.